builder/amazon/common/run_config.go

```diff
@@ -377,8 +377,20 @@ type RunConfig struct {
 	WindowsPasswordTimeout time.Duration `mapstructure:"windows_password_timeout" required:"false"`
 
 	// Communicator settings
-	Comm         communicator.Config `mapstructure:",squash"`
-	SSHInterface string              `mapstructure:"ssh_interface"`
+	Comm communicator.Config `mapstructure:",squash"`
+
+	// One of `public_ip`, `private_ip`, `public_dns`, or `private_dns`. If
+	//    set, either the public IP address, private IP address, public DNS name
+	//    or private DNS name will used as the host for SSH. The default behaviour
```

Comment:
```suggestion
	//    or private DNS name will be used as the host for SSH. The default behaviour
```

Reply:
Good catch. Pushed up a commit that fixes this and regenerates the code.